database/src/pool/sqlite.rs

```diff
@@ -332,7 +333,12 @@ impl Connection for SqliteConnection {
                                 .query_row(params![&sid, &cid.0], |row| row.get(0))
                                 .optional()
                                 .unwrap_or_else(|e| {
-                                    panic!("{:?}: series={:?}, aid={:?}", e, sid, cid);
+                                    // work around https://github.com/rusqlite/rusqlite/issues/777
+                                    if let rusqlite::Error::InvalidColumnType(_, _, rusqlite::types::Type::Null) = e {
+                                        None
+                                    } else {
+                                        panic!("{:?}: series={:?}, aid={:?}", e, sid, cid);
```

Comment:
Looking at that issue, it sounds like the way to do this is to query an Option<T> rather than T (i.e., adjust query_row's inferred return type to `rusqlite::Result<Option<T>>` or so).